### PR TITLE
Fix update_geneset to ensure functionnality

### PR DESCRIPTION
**Pull request by benjamin-millot**
Three main modifications are included in this PR:
- Rewrite update_genesets to correctly use crud.update_geneset.
- Rewrite update_geneset to correctly change a geneset in the database.
- Update README.md with better informations for install.

Please note:
- The URL to update a geneset is a PUT build like this one: http://127.0.0.1:8000/genesets/1 with a JSON body similar to this one:

```
{
    "title": "MY_CUSTOM_GENESET",
    "genes": [
        {"name": "aaa"}, 
        {"name": "bbb"}, 
        {"name": "ccc"}
    ]
}
```
- Empty `genes` are now supported

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -31,13 +31,13 @@ def read_match_genesets(pattern: str, db: Session = Depends(get_db)):
...
 @app.get("/genesets/{geneset_id}", response_model=schemas.Geneset)
 def read_geneset(geneset_id: int, db: Session = Depends(get_db)):
-
     return crud.get_geneset(db, geneset_id)
 
-@app.put("/genesets/{geneset_id}")
-def update_genesets(geneset_id: int, db: Session = Depends(get_db)):
 
-    return crud.get_geneset(db, geneset_id)
+@app.put("/genesets/update/{geneset_id}", response_model=schemas.Geneset)
```
Comment (LeonardSellam) [inline backend/main.py]: its not a really good practice to have the name of the method in the url. do we really need to have /update ?

**Review by LeonardSellam — CHANGES_REQUESTED** [review]
@benjamin-millot I think we should have a body instead of full url param for the update endpoint. Its much easier to deal with genes list. What do you think ? thks

File: backend/main.py (hunk 1 above)
Comment (benjamin-millot) [reply backend/main.py]: Nope, indeed. I wanted to follow the `search` scheme but actually that was a bad idea, I'm editing this.

**Comment by benjamin-millot** [comment]
Sure, I'll put the arguments in the body instead.

**Comment by benjamin-millot** [comment]
Everything should be updated now

File: backend/main.py [hunk 2]
```diff
@@ -31,13 +31,14 @@ def read_match_genesets(pattern: str, db: Session = Depends(get_db)):
...
-
     return crud.get_geneset(db, geneset_id)
 
-@app.put("/genesets/{geneset_id}")
-def update_genesets(geneset_id: int, db: Session = Depends(get_db)):
 
-    return crud.get_geneset(db, geneset_id)
+@app.put("/genesets/{geneset_id}", response_model=schemas.Geneset)
+def update_genesets(geneset_id: int, geneset: schemas.GenesetCreate, db: Session = Depends(get_db)):
+    print(geneset)
```
Comment (LeonardSellam) [inline backend/main.py]: @benjamin-millot  trailing log to remove :) thks

Comment (benjamin-millot) [reply backend/main.py]: Done!

**Review by LeonardSellam — APPROVED** [review]
Cool ! Thks a lot @benjamin-millot

**Comment by benjamin-millot** [comment]
👍

**Pull request merged**